ios/Podfile

```diff
@@ -36,41 +43,51 @@ abstract_target "ReactNative" do
 
   # Libs
   pod 'RNVectorIcons', :path => '../node_modules/react-native-vector-icons'
-  pod 'RNGestureHandler', :podspec => '../node_modules/react-native-gesture-handler/RNGestureHandler.podspec'
+  pod 'RNGestureHandler', :podspec => '../node_modules/react-native-gesture-handler'
+  pod 'RNReanimated', :podspec => '../node_modules/react-native-reanimated'
   pod 'react-native-sqlite-storage', :path => '../node_modules/react-native-sqlite-storage'
   pod 'react-native-calendar-events', :path => '../node_modules/react-native-calendar-events'
-  pod 'react-native-camera', :path => '../node_modules/react-native-camera'
+  pod 'react-native-camera', :path => '../node_modules/react-native-camera', subspecs: [
+    'TextDetector',
+    'FaceDetectorMLKit'
+  ]
   pod 'react-native-image-picker', :path => '../node_modules/react-native-image-picker'
   pod 'TouchID', :path => '../node_modules/react-native-touch-id'
   pod 'react-native-ble-plx', :path => '../node_modules/react-native-ble-plx'
   pod 'react-native-ble-plx-swift', :path => '../node_modules/react-native-ble-plx'
-  pod 'RNSound', :podspec => '../node_modules/react-native-sound/RNSound.podspec'
-  pod 'react-native-video', :podspec => '../node_modules/react-native-video/react-native-video.podspec'
+  pod 'RNSound', :podspec => '../node_modules/react-native-sound'
+  pod 'react-native-video', :podspec => '../node_modules/react-native-video'
   pod 'react-native-maps', :path => '../node_modules/react-native-maps'
   pod 'react-native-google-maps', :path => '../node_modules/react-native-maps'
   pod 'GoogleMaps'
   pod 'Google-Maps-iOS-Utils'
-  pod 'RNSVG', :path => '../node_modules/react-native-svg'
+  pod 'RNSVG', :podspec => '../node_modules/react-native-svg'
```

Comment:
Is this a conscious decision? Some of the pod spec define open-ended dependencies, which means it will download newer versions other than what we define in package.json. We had that before and therefor switched to paths.

Reply:
Unconscious mistake, fixed!